styleguide/components/21-atoms/link/_link.scss

```diff
@@ -6,37 +6,62 @@ a {
   text-decoration: underline;
   cursor: pointer;
 
+  &:hover {
+    @include theme('background-color', 'color-primary--lighten-4', 'link-background-hover');
+    @include theme('color', 'color-primary--darken-2', 'link-color-hover');
+  }
+
   &:focus {
     outline: thin dotted;
   }
 
-  //
-  // @TODO: Add hover state (waiting for design specifications.)
-  //
-  // &:hover {}
+  // Different button link types. These are links styled as a button.
+  &.button-link {
```

Comment:
Would change this to '.button' since the 'a' tag is already enforced above.
That way the default button class will always be overwritten in combination with a link, developers don't need to remember an extra class.

Reply:
Fixed it.